crypto/schemes.go

```diff
@@ -0,0 +1,110 @@
+package crypto
+
+import (
+	"crypto/cipher"
+
+	"github.com/drand/kyber"
+
+	bls "github.com/drand/kyber-bls12381"
+	"github.com/drand/kyber/pairing"
+	"github.com/drand/kyber/sign"
+
+	// FIXME package github.com/drand/kyber/sign/bls is deprecated: This version is vulnerable to
+	// rogue public-key attack and the new version of the protocol should be used to make sure a
+	// signature aggregate cannot be verified by a forged key. You can find the protocol in kyber/sign/bdn.
+	// Note that only the aggregation is broken against the attack and a later version will merge bls and asmbls.
+	signBls "github.com/drand/kyber/sign/bls"
```

Comment:
It's ok to use this package if you don't aggregate bls signature when verifying, which we don't , we only use the "native BLS signature" scheme, not the aggregated one. I'd either (1) remove this comment which may lead to think there is vuln while there is not in our usage OR (2) un-deprecate it but remove any methods that enbles to aggregate in kyber ? (if there are any)

Reply:
The text was already there in its previous location. I'm happy removing the comment, but maybe your second option is "cleaner". WDYT?

Reply:
I replaced the text with:

> The package github.com/drand/kyber/sign/bls is deprecated because it is vulnerable to rogue public-key attack against BLS aggregated signature. The new version of the protocol can be used to make sure a signature aggregate cannot be verified by a forged key. You can find the protocol in kyber/sign/bdn.
> Note that only the aggregation is broken by the attack and a later version will merge bls and asmbls.
> The way we are using this package does not do any aggregation and we're only using simple signatures and thus this is not a security issue for drand.